src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/commonplatform/PersonDetails.java

```diff
@@ -16,8 +19,12 @@
     private final String title;
     private final String firstName;
     private final String middleName;
+    @NotBlank
     private final String lastName;
+    @NotNull
     private final LocalDate dateOfBirth;
```

Comment:
The schema doesn't enforce `dateOfBirth` - should we relax this one ? We also allow it to be null in our database.